app.js

```diff
@@ -0,0 +1,72 @@
+//let mySchool = new School();
+const Class = require('./Class');
```

Comment:
You only needed to import the School class since school already has the other two classes imported on it.